Dockerfile

```diff
@@ -10,4 +10,4 @@ RUN ["chmod", "+x", "/wait"]
 
 RUN chmod +x /scripts/*.sh
 
-CMD "/wait" && "/scripts/create-jurisdiction.sh" && "/scripts/create-users.sh"
```

Comment:
Is create judistriction not needed? Have seen other related changes removed

Reply:
user-profile-url end point creates jurisdictions, so no need of separate script